includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -26,9 +23,30 @@ public function setHrefs( $hrefs ) {
 	 * @return string full URL, in case of lang specific URL missing, default one is returned
 	 */
 	public function getHref( $name, $lang ) {
+		if ( !isset( $this->hrefs[ $lang ] ) ) {
+			$lang = $this->getLangFallback( $lang );
+		}
+
 		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "default" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)